proposals/restructure-KF-docs-proposal.md

```diff
@@ -0,0 +1,227 @@
+  - Events (current About/Events Calendar)
+  - Google Summer of Code (keep)
+    
+- **Components** (remove "of Kubeflow")
+  - Jupyter Notebooks
+    - Overview of Jupyter Notebooks in Kubeflow
+    - Set Up Your Notebooks
+    - Create a Custom Jupyter Image
+    - Submit Kubernetes Resources
+    - Build a Docker Image on GCP
+    - Troubleshooting Guide
+  - Central Dashboard
+    - Central Dashboard 
+    - Registration Flow
+  - Metadata
+  - Fairing 
+    - Overview of Kubeflow Fairing
+    - Install Kubeflow Fairing
+    - Configure Kubeflow Fairing
+    - Fairing on Azure
+    - Fairing on GCP
+    - Tutorials
+    - Reference
+  - Feature Store
+    - Introduction to Feast
+    - Getting started with Feast
+  - Frameworks for Training
+    - Chainer Training
+    - MPI Training
+    - MXNet Training
+    - PyTorch Training
+    - TensorFlow Training (TFJob)
+  - Hyperparameter Tuning
+    - Introduction to Katib
+    - Getting started with Katib
+    - Running an experiment
+    - Katib Configuration Overview
+    - Environment Variables for Katib Components
+  - Pipelines
+    - Pipelines
+  - Tools for Serving
+    - Overview
+    - KFServing
+    - Seldon Core Serving
+    - BentoML
+    - NVIDIA Triton Inference Server
+    - TensorFlow Serving
+    - TensorFlow Batch Prediction
+  - Multi-Tenancy in Kubeflow
+    - Introduction to Multi-user Isolation
+    - Design for Multi-user Isolation
+    - Getting Started with Multi-user Isolation
+  - Miscellaneous
+    - Nuclio functions
+    
+- **Kubeflow Pipelines**
+  - Pipelines Quickstart
+  - Installing Pipelines
+    - Installation Options for Kubeflow Pipelines
+    - Kubeflow Pipelines Standalone Deployment
+    - Deploying Kubeflow Pipelines on a local cluster
+  - Understanding Pipelines
+    - Overview of Kubeflow Pipelines
+    - Introduction to the Pipelines Interfaces
+    - Concepts
+  - Building Pipelines with the SDK
+    - Introduction to the Pipelines SDK
+    - Install the Kubeflow Pipelines SDK
+    - Build Components and Pipelines
+    - Create Reusable Components
+    - Building Python function-based components
+    - Best Practices for Designing Components
+    - Pipeline Parameters
+    - Python Based Visualizations
+    - Visualize Results in the Pipelines UI
+    - Pipeline Metrics
+    - DSL Static Type Checking
+    - DSL Recursion
+    - Using environment variables in pipelines
+    - GCP-specific Uses of the SDK
+    - Manipulate Kubernetes Resources as Part of a Pipeline
+  - Multi-user Isolation for Pipelines
+  - Caching
+  - Upgrading
+  - Samples and Tutorials
+    - Using the Kubeflow Pipelines Benchmark Scripts
+    - Using the Kubeflow Pipelines SDK
+    - Experiment with the Kubeflow Pipelines API
+    - Experiment with the Pipelines Samples
+    - Run a Cloud-specific Pipelines Tutorial
+  - Troubleshooting
+  - Reference
+    - Component Specification
+    - Pipelines API Reference
+    - Pipelines SDK Reference
+- **Kubeflow on AWS**
```

Comment:
Should `AWS`, `Azure`, `GCP`, `IBM Cloud` and `Openshift`docs live under a `Platforms` section?